app/src/androidTest/java/com/github/leuludyha/ibdb/presentation/components/author_view/FullAuthorViewTest.kt

```diff
@@ -49,4 +49,13 @@ class FullAuthorViewTest {
             .assertExists("A liked work should appear in the reading list")
     }
 
+    @Test
+    fun clickingOnWorkOpenBookView() {
```

Comment:
It throws an exception because the `NavGraph` can't be correctly initialised in the tests. However the name of the test is misleading sorry about that I forgot to change it